test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/OutputConsoleLoggerTests.Log.cs

```diff
@@ -33,13 +40,13 @@ public void Non_integer_verbosity_is_interpreted_as_2()
 
             [Theory]
             [MemberData(nameof(GetMessageVariationsWhichAreLogged))]
-            public void Writes_line_to_output_console(LogLevel logLevel, int verbosityLevel)
+            public async void Writes_line_to_output_console(LogLevel logLevel, int verbosityLevel)
             {
                 _outputConsole.Reset();
                 _msBuildOutputVerbosity = verbosityLevel;
 
                 _outputConsoleLogger.Log(new LogMessage(logLevel, "message"));
-
+                await WaitForInitialization();
```

Comment:
Anywhere I've added this wait is a test I observed failing when I introduced a `Task.Delay` in the product code. It's possible other tests need this that I didn't catch, but at least there's code to refer to now.